airbyte-integrations/connectors/destination-postgres/src/main/java/io/airbyte/integrations/destination/postgres/PostgresSqlOperations.java

```diff
@@ -24,6 +26,19 @@ public PostgresSqlOperations() {
     super(new PostgresDataAdapter());
   }
 
+  @Override
+  protected List<String> postCreateTableQueries(final String schemaName, final String tableName) {
+    if (TypingAndDedupingFlag.isDestinationV2()) {
+      return List.of(
+          "CREATE INDEX IF NOT EXISTS " + tableName + "_raw_id" + " ON " + tableName + "(_airbyte_raw_id)",
+          "CREATE INDEX IF NOT EXISTS " + tableName + "_extracted_at" + " ON " + tableName + "(_airbyte_extracted_at)",
+          "CREATE INDEX IF NOT EXISTS " + tableName + "_loaded_at" + " ON " + tableName + "(_airbyte_loaded_at, _airbyte_extracted_at)"
+      );
+    } else {
+      return Collections.emptyList();
+    }
+  }
```

Comment:
> There doesn't seem to be any way of specifying an index in the CREATE TABLE syntax. 

[TIL](https://stackoverflow.com/questions/6239657/can-you-create-an-index-in-the-create-table-definition)!

Reply:
There are [all sorts if indexes](https://www.postgresqltutorial.com/postgresql-indexes/postgresql-index-types/) we could be creating (Btree, hash, etc), but I think "unspecified" index type like this is the right thing to do, as it will fallback to the default for the database/schema

Reply:
It would also be reasonable to make the index on `_raw_id` UNIQUE, but we make this field and we can trust UUID to be unique, so I don't think we need this validation.  Adding a UNIQUE index would also make writes slower, and we don't want that.

Reply:
these are good thoughts which I definitely did not think :P

I'll copypaste it into our code as an actual comment for easier future reference